CleanWeather/Support/en.lproj/Localizable.strings

```diff
@@ -24,7 +24,12 @@
 "timeMachine" = "Time Machine";
 "now" = "Now";
 "favouriteCities" = "Favourite Cities";
+"all" = "All";
+"favourites" = "Favourites";
```

Comment:
you could think about doing these localizable strings in a nicer way - but in a separate PR

I am used to:

"FavouriteCities.Title" = "Favourite Cities";
"FavouriteCities.Favourites = "Favourites";
"FavouriteCities.All = "All";
"FavouriteCities.SearchPlaceholder = "Search Cities";

etc